tabs-tabs-tabs/tabs.js

```diff
@@ -1,23 +1,39 @@
+function firstUnpinnedTab(tabs) {
+  for (var tab of tabs) {
```

Comment:
This will include tabs in all windows, so it's not going to give you what you expect when multiple windows are open.